components/profile_traits/time.rs

```diff
@@ -46,6 +46,11 @@ pub enum ProfilerMsg {
     ),
     /// Message used to force print the profiling metrics
     Print,
+
+    // TODO pylbrecht
+    // write meaningful docstring
```

Comment:
/// Report a layout query that could not be processed immediately for a particular URL.